src/helm/benchmark/static/schema.yaml

```diff
@@ -2827,6 +2827,26 @@ run_groups:
       when: n/a
       language: synthetic
 
+## Financial scenarios
+  - name: fin_qa
+    display_name: FinQA
+    short_display_name: FinQA
+    description: FinQA is a question answering task over financial reports that requires robust numerical reasoning [(Chen et al., 2021)](https://arxiv.org/abs/2109.00122).
+    metric_groups:
+      - accuracy
+      - calibration
+      - general_information
+    environment:
+      main_name: f1_score
+      main_split: test
+    taxonomy:
+      task: question answering
+      what: "?"
+      who: "?"
+      when: "?"
```

Comment:
Can we fill these in?  (We really should go back and fill these in for all the scenarios...)

Reply:
can't do it for this one (no real hf dataset) but could auto-fill missing ones with data cards?